modules/server/src/main/ts/bedrock/compiler/Webpack.ts

```diff
@@ -204,6 +204,12 @@ const getWebPackConfigJs = (scratchFile: string, dest: string, coverage: string[
         ] : []
       )
     },
+    plugins: [
+      new webpack.DefinePlugin({
```

Comment:
Should we be applying this in TS mode as well? I realise you only need it for JS mode, but I'm not too keen on diverging too much between the 2 modes.

Reply:
JS mode is really only used by RTC, so I was aiming to make it as small a change as possible, but I can add it to both.

Reply:
I just figured it shouldn't hurt anything to have it enabled by default anyways :shrug: